operations/repo.go

```diff
@@ -113,26 +116,40 @@ func repoSubmit() cli.Command {
 				Usage:  "specify the API key to authenticate to the repobuilding service",
 				EnvVar: "BARQUE_API_KEY",
 			},
+			cli.StringFlag{
```

Comment:
Is there a reason these environment variable names need to be configurable? I feel like the environment variable names should be fixed.

Reply:
It will make it easier for non-server releases (like `mongo-tools`) that are on different release cadences and want to release the same version of their project to different server versions. This happens in the same push task, so switching around env vars can get messy.

Reply:
It works well for mongo cause each version has its own branch, so in the evg config the correct key is stored as the default env var.